tests/e2e/performance/csi_tests/test_pvc_snapshot_performance.py

```diff
@@ -426,6 +529,34 @@ def test_pvc_snapshot_performance_multiple_files(
 
         all_results = []
 
+        self.full_log_path = get_full_test_logs_path(cname=self)
+        self.results_path = get_full_test_logs_path(cname=self)
+        if interface == constants.CEPHBLOCKPOOL:
+            self.sc = "RBD"
+        elif interface == constants.CEPHFILESYSTEM:
+            self.sc = "CephFS"
+        elif interface == constants.CEPHBLOCKPOOL_THICK:
+            self.sc = "RBD-Thick"
+        self.full_log_path += f"-{self.sc}-{interface}"
```

Comment:
this should be : `self.full_log_path += f"-{file_size}-{files}-{threads}-{interface}"`